src/Standards/AcquiaPHP/ruleset.xml

```diff
@@ -8,23 +8,18 @@
 
   <description>Acquia's PHP coding standards.</description>
 
+  <rule ref="PSR12">
+    <exclude name="Generic.WhiteSpace.ScopeIndent"/>
+    <exclude name="Generic.PHP.LowerCaseConstant"/>
+  </rule>
+
   <!-- Drupal sniffs -->
-  <!-- These provide PHPCBF integration, unlike their Squiz equivalents -->
```

Comment:
I think the Squiz versions have had PHPCBF integration for like... 10 years? Not sure how old this comment is by comparison 😂 

https://github.com/squizlabs/PHP_CodeSniffer/commit/e560da9a52c01472a42d8854e8bb987f22348478